### DISCUSSION TITLE
Fail to run run_tests in latest master (commitID = b3305e18e7adf62db58d8f0dc4b075772201a35c)

### DISCUSSION THREAD

**Discussion by one-step-per-week** [idaholab/moose#24323] [Q&A Getting Started] [answered]
Error message:
`Error! Could not find libmesh_config.h in any of the usual locations!`

I am using a docker with a clean ubuntu 22.04 image following this Conda-way installation: [installation Steps](https://mooseframework.inl.gov/getting_started/installation/conda.html). I could be able to run the run_tests yesterday, but 5 of them failed. Today I want to rerun the tests, but the above error occurred.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did you remember to activate the conda environment?

Guillaume

**Reply by one-step-per-week** [reply]
Thanks for your reply. There is a "(moose)" prefix in front of my terminal prompt, I guess it is the correct conda environment you mentioned?

**Reply by one-step-per-week** [reply]
@GiudGiud My bad... I just found that I didn't execute the "make -j 6"... what a silly mistake! Now it can run. Thank you for your time.
